source/_posts/2015-12-21-wye-tech-news-3.markdown

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title: "Wye Tech News #3"
+category:
+comments: true
+author:
+  name: Jorge Bejar
+  email: jorge@wyeworks.com
+  twitter_handle: jmbejar
+  github_handle:  jmbejar
+  image:  /images/team/jorge-bejar.jpg
+  description: Software Engineer at Wyeworks. Ruby on Rails developer.
+---
+
+Hey it's Monday! So here is a new issue of our weekly compilation of interesting news and articles about Ruby, Rails, JavaScript and web development related stuff.
```

Comment:
What about rephrasing it to something along these lines? "Hey! It's Monday! You know what that means? It's time for a new issue of..."